application/backend/src/main/java/io/typerefinery/websight/models/components/layout/Container.java

```diff
@@ -239,12 +256,25 @@ public String[] getClasses() {
     @Override
     @PostConstruct
     protected void init() {
-        this.id = DEFAULT_ID;
         this.module = DEFAULT_MODULE;
         super.init();
 
         String flex = "";
 
+
+        
+
+        if(StringUtils.isBlank(defaultPadding)) {
+            defaultPadding = "enabled";
+        }
+
+        if(defaultPadding == "enabled") {
```

Comment:
defaultPadding = DEFAULT_CONTAINER_CLASSES?

Do you just want to make this a default class and remove grid.addClasses(resource.getName());?

Reply:
or does defaultPadding  should be called something more appropirate?

Reply:
If we do something like this resiource.getName(), for container It will return a "container", so bootstrap has a class called container; this will basically have some padding.

so I had a variable called defaultPadding, 
so when that is enabled, "container" will be added; 
if it is disabled, then "container-fluid" will be added, which takes 100% width without any padding.

Reply:
So that check box you are choosing if this container will be "Full Width" or not?